app/src/renderer/connectors/rpcWrapper.js

```diff
@@ -24,15 +23,15 @@ module.exports = function setRpcWrapper(container) {
 
       rpcWrapper.rpcInfo.connected = false
     },
-    rpcConnect(nodeIP) {
-      rpcWrapper.rpcInfo.nodeIP = nodeIP
+    rpcConnect(rpcURL) {
+      let rpcHost = rpcURL.startsWith(`http`) ? rpcURL.split(`//`)[1] : rpcURL
```

Comment:
this will throw if the URL starts with `http` but doesn't contain `//`, we should test both cases

Reply:
which url starts with `http` but doesn't contain `//`?